### DISCUSSION TITLE
Does it support automated testing of 3D models in WebGL?

### DISCUSSION THREAD

**Discussion by luckyboy-wei** [seleniumbase/SeleniumBase#3378] [Q&A] [answered]
Does it support automated testing of 3D models in WebGL? If not supported, is there any way to recommend it?

**Comment by mdmintz** [comment, marked as answer]
Set `enable_3d_apis=True` to enable WebGL and 3D APIs.

Example: https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_robot.py
https://github.com/seleniumbase/SeleniumBase/blob/11e81b008f0c133e4c6e27863293901e8669e1d5/examples/raw_robot.py#L3

Also see: https://github.com/seleniumbase/SeleniumBase/blob/master/examples/test_3d_apis.py
